spec/features/categories_home_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'Categories home ' do
+  include Devise::Test::IntegrationHelpers
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly you have few failed `specs`, all specs should pass so please fix them 🙏🏼 

![image](https://github.com/ichsansandy/budget-buddy/assets/63915024/72beffad-0281-4fe2-a597-e279f7c0b766)

Reply:
Passing